### ISSUE TITLE
memory leak

### ISSUE THREAD

**Issue by urekalion** [dsindex/segm-lstm#2]
inference.py 에서 memory leak이 있습니다.

`result = sess.run(tf.arg_max(logits, 1), feed_dict=feed)`
를
`argmax = tf.arg_max(logits, 1)`
`result = sess.run(argmax, feed_dict=feed)`
로 수정하니 해결되었습니다 :)

**Comment by dsindex** [comment]
@urekalion 
오 그런게 있군요. 감사합니다 ^^
